dev/com.ibm.ws.jpa.container.core/resources/com/ibm/ws/jpa/jpa.nlsprops

```diff
@@ -344,3 +344,8 @@ PROVIDER_FATAL_CWWJP9993E.explanation=The Java Persistence API (JPA) provider ha
 PROVIDER_FATAL_CWWJP9993E.useraction=See the JPA provider documentation.
 
 #-------------------------------------------------------------------------------
+FAILED_TO_CREATE_PU_BEAN_CWWJP9994W=CWWJP9994W: Could not create bean for PersistenceUnit {0}, it will not be injectable
```

Comment:
Rewrite as:
CWWJP9994W: Could not create bean for the {0} persistence unit. It is not injectable.